### ISSUE TITLE
How to get the polarity score with this library?

### ISSUE THREAD

**Issue by rafw** [cdipaolo/sentiment#7]
Just like python textblob library (but it's too slow and hard to embedded in go), using [-1, 1] float number to describe sentiment, -0.x means bad, +0.x means good, -1 means worst, 1 means best.

Thanks a lot!

**Comment by cdipaolo** [comment]
tl;dr: you can use ``model.SentimentAnalysis("You're mother is an awful lady", sentiment.English) * 2 - 1`` to get the score in ``[-1,1]`` and cast to float as needed. That said, this package only returns `{0,1}` by default so you'll only get `{-1,1}`.

When I wrote this I calculated a binary score, whether the sentence is more likely to be positive (`1`) or negative (`1`). This was done for computational reasons, and while theoretically you could return probabilities, I didn't back when I wrote the library [more specifically the goml/text package] and it would take a _bit_ of elbow grease to build it. I don't have the time right now to focus on this but if you'd like to implement the functionality go for it. I can give more details if you'd like.
